### PR TITLE
New: Rules via cli

### LINKED ISSUE #1020: Support define rules via CLI (would ignore .sonarwhalrc rules)
Allow

```
sonarwhal https://example.com --rules http-cache manifest-is-valid ssllabs
```

that would skip .sonarwhalrc rules configuration and apply only rules **http-cache**, **manifest-is-valid** and **ssllabs** with its default configuration (`rule: "error"`).

The benefit: running all defined rules in .sonarwhalrc may be very slow and, while working on website fixes for specific role, it would be easier, quicker and "directly to target" to revalidate by single/few rule/s defined in CLI.
Since it might be complicated to defined custom config per rule in CLI (maybe like `sonarwhal https://example.com --rules ssllabs:["error", {"grade": "A+"}]`; also applying .sonarwhalrc rule config from  would be wrong here since actually wanted to ignore it here), I suggest currently apply the rules default configuration.

### PR DESCRIPTION
**Pull request by kshyju**
This change will allow users to explicitly pass the rules via cmd line.

Example : `sonarwhal https://example.com --rules http-cache,html-checker`

Fix #1020

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messagess)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,12 @@ To use a different formatter than the one specified in your `.sonarwhalrc` file
 sonarwhal https://example.com --formatters excel
 ```
 
+To use a different rule than the one specified in your `.sonarwhalrc` file:
+
+```bash
+sonarwhal https://example.com --rules html-checker
+```
+
```
Comment (alrra) [inline README.md]: Maybe we should also add an example for using multiple rules?

File: packages/sonarwhal/src/lib/config.ts [hunk 2]
```diff
@@ -129,6 +142,14 @@ const updateConfigWithCommandLineValues = (config: UserConfig, actions: CLIOptio
         config.formatters = actions.formatters.split(',');
         debug(`Using formatters option provided from command line: ${actions.formatters}`);
     }
+
+    // If rules are provided, use them
+    if (actions && actions.rules) {
+        const rulesarray = actions.rules.split(',');
+
+        config.rules = buildRulesConfigFromRuleNames(rulesarray, 'warning');
```
Comment (alrra) [inline packages/sonarwhal/src/lib/config.ts]: Shouldn't we use `error`?

Personally find useful to have the right status code, even for quick things.

File: README.md (hunk 1 above)
Comment (kshyju) [reply README.md]: Updated the readme with a multi rule example. Perhaps we should keep the read me with simple ones and create a wiki page with complete set of examples/options and use cases

File: packages/sonarwhal/src/lib/config.ts (hunk 2 above)
Comment (kshyju) [reply packages/sonarwhal/src/lib/config.ts]: Most of the time, I use `warning` for rules,  as i did not want the program to exit. If you have a strong opinion it should be `error`, i will change it.

Comment (kshyju) [reply packages/sonarwhal/src/lib/config.ts]: Switched to `error`.

File: README.md (hunk 1 above)
Comment (alrra) [reply README.md]: >  Perhaps we should keep the read me with simple ones 

We could consider moving the information into its own dedicated page.

**Pull request closed without merging**